### PR TITLE
Add MI-PYT B181

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: runs/2018/mipyt-zima/info.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  date: 2018-11-28
+
+- base: async
+  date: 2018-12-05
+
+- base: magic
+  date: 2018-12-12
+
+- base: upy
+  date: 2018-12-19
```
Comment (hroncok) [inline runs/2018/mipyt-zima/info.yml:51]: @encukou Would you be able to come? It's Pyvo date. We _need you_ (and your hardware) for MicroPython and we can possibly shuffle around async/magic/upy.

File: runs/2018/mipyt-zima/info.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+derives: mi-pyt
+plan:
+- base: requests-click
+  date: 2018-10-03
+
+- base: flask
+  date: 2018-10-10
+
+- base: pandas
+  date: 2018-10-17
```
Comment (hroncok) [inline runs/2018/mipyt-zima/info.yml:24]: @encukou Would you be able to come? It's Pyvo date. We _can_ deliver this without you, but I feel that you still do it a little bit better.

File: runs/2018/mipyt-zima/info.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- base: magic
+  date: 2018-12-12
+
+- base: upy
+  date: 2018-12-19
+
+- slug: extra
+  title: Konzultace
+  date: 2019-01-02
```
Comment (hroncok) [inline runs/2018/mipyt-zima/info.yml:55]: One day after New Years, I've decided that we just don't do anything much complicated.

File: runs/2018/mipyt-zima/info.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  date: 2018-10-31
+
+- base: testing
+  date: 2018-11-07
+
+- base: cython
+  date: 2018-11-14
+
+- base: docs
+  date: 2018-11-21
```
Comment (hroncok) [inline runs/2018/mipyt-zima/info.yml:39]: @encukou Would you be able to come? It's Pyvo date. We are absolutely OK to deliver this ourselves. But if you'd like it, it's yours.

File: runs/2018/mipyt-zima/info.yml (hunk 3 above)
Comment (hroncok) [reply runs/2018/mipyt-zima/info.yml:55]: TODO: next year we should throw in a Python 2 farewell party

File: runs/2018/mipyt-zima/info.yml (hunk 1 above)
Comment (encukou) [reply runs/2018/mipyt-zima/info.yml:51]: Yes.
30 hardwares, right?

Comment (hroncok) [reply runs/2018/mipyt-zima/info.yml:51]: 24 working ones would do. 30 if we include some extras for sure.

File: runs/2018/mipyt-zima/info.yml (hunk 4 above)
Comment (encukou) [reply runs/2018/mipyt-zima/info.yml:39]: Here PyLadies need me for MicroPython, so I won't come to Prague at all.

File: runs/2018/mipyt-zima/info.yml (hunk 2 above)
Comment (encukou) [reply runs/2018/mipyt-zima/info.yml:24]: OK

File: runs/2018/mipyt-zima/info.yml (hunk 4 above)
Comment (hroncok) [reply runs/2018/mipyt-zima/info.yml:39]: ack

**Review by encukou — APPROVED** [review]

**Review by MarekSuchanek — APPROVED** [review]

**Pull request merged**